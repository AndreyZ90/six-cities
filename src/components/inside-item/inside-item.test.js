import React from 'react';
import TestRenderer from 'react-test-renderer';

import InsideItem from '@/components/inside-item/inside-item';

const item = `Wi-Fi`;

describe(`InsideItem component snapshot`, () => {
  test(`Should correctly render InsideItem component`, () => {
    const tree = TestRenderer.create(<InsideItem item={item} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
