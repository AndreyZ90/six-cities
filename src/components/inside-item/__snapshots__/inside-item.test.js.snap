// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsideItem component snapshot Should correctly render InsideItem component 1`] = `
<li
  className="property__inside-item"
>
  Wi-Fi
</li>
`;
